utils/polus-imagej-util/{{cookiecutter.project_slug}}/push-docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+version=$(<VERSION)
+docker push polusai/imagej-sobel-plugin:${version}
```

Comment:
This needs some jinja code. Replace `imagej-sobel-plugin` with a jinja variable referencing the plugin name.